src/chapters/14-provider-components.md

```diff
@@ -294,10 +288,12 @@ In our `<Rentals>` component, we used the `as |results|` syntax when invoking `<
 
 > Zoey says...
 >
-> The local variable name `results` is arbitrary, and isn't special in any way! You could name it anything: `as |data|`, `as |filtered|`, or even `as |banana|`! Really, the important here is that however you name the block param is how you will have access to the yielded data from inside of the block.
+> The local variable name `results` is arbitrary, and isn't special in any way! You could name it anything: `as |data|`, `as |filtered|`, or even `as |banana|`! Really, the important thing here is that however you name the block param is how you will have access to the yielded data from inside the block.
 
 Interestingly, if we take a look at our `<Rentals::Filter>` component template, we see that we don't actually render any content. Instead, this component's only responsibility is to set up some piece of state (`this.results`, the list of filtered rental properties), and then yield that state back up to its caller (`<Rentals>`) in the form of a block parameter (`as |results|`).
 
+This is called the (provider component pattern)[TODO: link to provider component pattern], which we see in action with one component providing data up to its caller.
```

Comment:
```suggestion
This is called the *(provider component pattern)[TODO: link to provider component pattern]*, which we see in action with one component providing data up to its caller.
```

Something about the "..., which we see in action..." part reads a bit confusing to me. 🤔 I think it may be fine to just drop that second half and merge the sentence into the previous paragraph?